BACKEND/src/config.js

```diff
@@ -1,12 +1,13 @@
-//zmienić później na dotenv
-const config = {
+import * as dotenv from 'dotenv';
+const __dirname = import.meta.dirname;
+dotenv.config({path: `${__dirname}/config.env`});
+
+export const config = {
     db: {
-      host: "127.0.0.1",
-      user: "root",
-      password: "QweAsdZxc2203#",
```

Comment:
Whoa haslo do bazy ktore nie jest "superuser". Hackers FEAR THIS MAN

Reply:
xDDD, @Tesa44 ale to jest chyba do lokalnej bazy, a mamy już postawioną online, więc bez obaw 😎